### ISSUE TITLE
Problem z parametrem niejednoznacznym

### ISSUE THREAD

**Issue by goses** [allegro/allegro-api#10411]
Witam,
Dawno nie dodawaliśmy ofert z nowymi produktami i z parametrami "Producent części -> inny" oraz "Marka maszyny -> Inna", ale teraz mamy parę takich produktów, jednak nie możemy ich dodać bo wyświetla się błąd w naszym skrypcie **Nieprawidłowe wartości parametrów: [parameter = Marka maszyny [ Wartość niejednoznaczna - Inna]]** . Z tego co pamiętam to dodawaliśmy wcześniej takie produkty bez problemu... Parametr "Producent części -> inny" wymaga podania dodatkowego pola Value, który podajemy, "Marka maszyny -> Inna" nie wymaga tego, co dziwne inne oferty które mają dodany produkt do katalogu, bądź z wybranym z listy Producentem części ale "Marka maszyny -> Inna" przechodzą bez problemu.

Parametry produktu które wysyłamy:
```
[parameters] => Array
        (
            [0] => Array
                (
                    [id] => 215858
                    [values] => Array
                        (
                            [0] => 563827-Z
                        )

                )

            [1] => Array
                (
                    [id] => 227345
                    [values] => Array
                        (
                            [0] => 563827
                        )

                )

            [2] => Array
                (
                    [id] => 128491
                    [valuesIds] => Array
                        (
                            [0] => 128491_32
                        )

                )

            [3] => Array
                (
                    [id] => 247835
                    [valuesIds] => Array
                        (
                            [0] => 247835_821153
                        )

                    [values] => Array
                        (
                            [0] => Agro-Serwis
                        )

                )

            [4] => Array
                (
                    [id] => 128590
                    [valuesIds] => Array
                        (
                            [0] => 128590_2097152
                        )

                )

        )
```

Kategoria:
```
 [category] => Array
        (
            [id] => 252248
        )
```

Szczegóły błędu:
```
array (
  'errors' => 
  array (
    0 => 
    array (
      'code' => 'ProductValidationException',
      'message' => 'Nieprawidłowe wartości parametrów: [parameter = Marka maszyny [ Wartość niejednoznaczna - Inna]]',
      'details' => NULL,
      'path' => 'productSet[0].product',
      'userMessage' => 'Nieprawidłowe wartości parametrów: [parameter = Marka maszyny [ Wartość niejednoznaczna - Inna]]',
      'metadata' => 
      array (
      ),
    ),
  ),
) 
```

trace-id: 98af8453742caceb

**Comment by goses** [comment]
Hm, chyba nasze wszystkie oferty mają zaznaczoną Markę maszyny jako "Inna" a jak już jakieś są to pojedyncze przypadki. A wszystkie oferty były wystawiane przez nasz skrypt... i ta wartość "Inna" jest z listy słownikowej pobieranej z waszego API. Nawet wczoraj były dodawane oferty z takim ustawieniem ale teraz nie potwierdzę innych ustawień. W takim konkretnym przypadku gdzie ofertę wystawiamy z nowym produktem i z parametrem "Producent części -> inny" oraz "Marka maszyny -> Inna" to były wystawiane na pewno w sierpniu i wrześniu, tutaj przykład takiej oferty: https://allegro.pl/oferta/196007z-zamiennik-manitou-wentylator-chlodnicy-silnika-16738354762

**Comment by Lukasz-Zurek** [comment]
Zgadza się. Skonsultowałem to jednak z odpowiednim zespołem i obecnie nie przyjmiemy wartości "inna", a nie ma możliwości przekazania dla niej własnej wartości (“customValuesEnabled”: false). Konieczne jest skorzystanie z listy naszych dostępnych wartości słownikowych. Jeśli preferowanej przez Ciebie marki nie ma, sprzedający ma możliwość zgłoszenia brakującej marki poprzez [ten formularz](https://salescenter.allegro.com/parameter-value-suggestions/create-suggestion) dostępny w salescenter (w API nie udostępniamy takiej funkcjonalności).

**Comment by goses** [comment]
No to tym samym ograniczacie możliwość dodawania ofert, mam nadzieję że to się szybko zmieni, powinniście odblokować taką możliwość tak jak było wcześniej, albo dać możliwość wpisania customValue... tak jak to jest dla Producenta części... 

Przez ten formularz chyba coś kiedyś zgłaszaliśmy ale jest mało praktyczny, teraz też chciałem dodać ale tylko dla jednej kategorii mogę to zgłosić a przecież jest sporo kategorii gdzie brakuje tej Marki (jak i wielu innych)... ale zgłaszanie wszystkiego pojedynczo jest uciążliwe i tylko traci się czas.

![Marka Maszyny Manitou ALLEGRO 1](https://github.com/user-attachments/assets/80a68f58-aeb4-444f-99bf-f8624ecbaacb)

**Comment by goses** [comment]
Ok... w takim razie zgłosiłem to przez formularz kontaktowy, zobaczymy co z tym zrobią. Pozdrawiam
